ubuntu/R/README.md

```diff
@@ -0,0 +1,21 @@
+# R container
+
+This image shows how to setup R and RStudio Server. 
+
+Note that you will still need to use an init script to start the RStudio Server
+daemon. Here is an example notebook python cell that installs an init script
+on a DBFS location.
+
+```py
+script = """#!/bin/bash
+set -euxo pipefail
+RSTUDIO_BIN="/usr/sbin/rstudio-server"
+
+if [[ ! -f "$RSTUDIO_BIN" && $DB_IS_DRIVER = "TRUE" ]]; then
+  rstudio-server restart || true
+fi
+"""
```

Comment:
did you know you can bake init scripts into the container? https://databricks.atlassian.net/browse/DOC-1120